README.md

```diff
@@ -122,7 +136,7 @@ minutes in our case). If the lock is already being held by other worker. This es
 used to process other messages in the queue.
 
 So we introduced an optional behavior which offers a Non-Blocking acquire lock implementation. While trying to acquire
-lock, The client can now optionally set ```shouldSkipBlockingWait = true ``` to prevent the user thread from being
+lock, the client can now optionally set ```shouldSkipBlockingWait = true ``` to prevent the user thread from being
```

Comment:
Inline prefirmatted text can be wrapped with a single backtick.

Reply:
Sure. I didn't add it, but can change it of course ;-)